projects/vassily-kandinsky/index.md

```diff
@@ -0,0 +1,156 @@
+
+In **1911** he and Franz Marc had founded *The blue rider* and organised international avant-garde exhibitions in Munich. They develop to the most important exhibitions of the German expressionists.
+
+During the **First World War** he lived in Russia where he had taught and organised other art exhibitions. 
+In **1921** he returned to Germany and started to taught at the Bauhaus school in Weimar. 
+Kandinsky published in **1926** his work *'point and line to plane'*. In it he propagates for an abstract *absolute* painting art. This work becomes an other important basis for the creative process within the abstract art. 
+His time ends there in **1933** with the closing of the school by the National Socialists. 
+
+After the Bauhaus period Kandinsky moves to France. There he lives and paints up to his death in **1944** in a Paris suburb. He had become a member of the Parisians artist's group Abstraction-Création. From the National Socialists Kandinskys pictures were devaluated, like from many other artists in this time, they had become „degenerated art“. Many of his pictures are exiled in **1937** from the German museums.
+
+## Inspiration
+**'Point and line to plane'** 
+In this varied work he describes the psychological effects of the colour and the resemblances between painting and music. This book had become a fundamental importance for the abstract painting. In the most important chapter „Forms and colour language“ he develops his theory that form can exist in contrast to colour independently. 
+That means that the form influences the colour substantially. 
+ 
+This work had woken up immediately my interest. They are so simple and as well so appropriate, although it concerns things which are rather unconsciously and true. 
+
+For example, he describes the **point** as such in the everyday use. How his primary meaning, in an assembly-line text, is taken from him and no one perceives the point anymore as a circle. 
+
+Then he describes **music instruments** with colours, for example is yellow a trumpet. In the beginning the trumpet is stimulating and nice, however, the longer you listen to her play, the more aggressively and more unbearably becomes the sound. 
+This attribute, of the sound and the glare, fits perfectly to the sharp triangle. 
+In the beginning **this book** was my basis, nevertheless, finally I have decided to take a certain work for the analysis (deeply). It is the painting *Circles in a Circle* of 1923. This was one of his first paintings in which he had occupied only with the circle.
+
+## Part 2 - Task
+- Analyse the artist after categories and gestalt laws. 
+- Besides, we should sketch methodologies and tools for the analysis.
+- According to the analysis we should formulate a set of rules and apply it in code.
+ 
+## Analysis - Circles in a Circle 1923
+![](assets/images/circles-in-a-circle.png)￼
+
+###### Source: http://www.wassilykandinsky.net/work-247.php | adapted to image size
```

Comment:
On the website I didn't found something about the license but wikipedia had used it as well from there. so I think it's okay? I found that on wikipedia.
https://commons.wikimedia.org/wiki/Commons:Reuse_of_PD-Art_photographs#Germany